cyral/resource_cyral_integration_idp_saml.go

```diff
@@ -93,9 +105,21 @@ func resourceIntegrationIdPSAML() *schema.Resource {
 		Description: "Manages identity provider (IdP) integrations using SAML to allow " +
 			"[Single Sing-On](https://cyral.com/docs/sso/overview) to Cyral.\n\nSee also " +
 			"the remaining SAML-related resources and data sources.",
-		CreateContext: CreateResource(
-			CreateGenericSAMLConfig(),
-			ReadGenericSAMLConfig(),
+		CreateContext: CRUDResources(
+			[]ResourceConfig{
+				{
+					Type:            create,
+					OperationConfig: CreateGenericSAMLConfig(),
+				},
+				{
+					Type:            read,
+					OperationConfig: ReadGenericSAMLConfig(),
+				},
+				{
+					Type:            update,
```

Comment:
Based on my previous comment, I believe we should just consider this as an create operation, since an actual resource is being created. Does that make sense?
```suggestion
					Type:            create,
```